demos/browser/server.js

```diff
@@ -44,11 +40,18 @@ if (captureS3Destination) {
   console.info(`S3 destination for capture not set.  Cloud media capture will not be available.`)
 }
 
+function checkValidControlRegion(controlRegion){
+  return (controlRegion == 'us-east-1' || controlRegion == 'us-west-2' || controlRegion == 'eu-central-1' || controlRegion == 'ap-southeast-1');
+}
+
 // return regional API just for Echo Reduction for now.
-function getClientForMeeting(meeting) {
-  if (meeting && meeting.Meeting && meeting.Meeting.MeetingFeatures && meeting.Meeting.MeetingFeatures.Audio &&
+function getClientForMeeting(meeting, controlRegion) {
+  if (checkValidControlRegion(controlRegion)){
+    return (new AWS.ChimeSDKMeetings({ region: controlRegion }));
+  } 
+  else if (meeting && meeting.Meeting && meeting.Meeting.MeetingFeatures && meeting.Meeting.MeetingFeatures.Audio &&
     meeting.Meeting.MeetingFeatures.Audio.EchoReduction === 'AVAILABLE') {
-      return chimeRegional;
+      return (new AWS.ChimeSDKMeetings({ region: 'us-east-1' }));
```

Comment:
Is `us-east-1` the correct region here?  This looks like it should be
```
return new AWS.ChimeSDKMeetings({ region: controlRegion });
```

Reply:
Actually no as it would come to this condition only if controlRegion is not valid.